src/main/java/org/example/cosmocats/featuretoggle/FeatureToggles.java

```diff
@@ -0,0 +1,15 @@
+package org.example.cosmocats.featuretoggle;
+
+import lombok.Getter;
+
+@Getter
+public enum FeatureToggles {
```

Comment:
I think you a bit confused why this needed. Please rewatch lecture, if you have questions - you may ask in the chat or on the lecture